pages/about/policies/criticalincidentmanagement.html

```diff
@@ -1 +1,1332 @@
-
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
```

Comment:
@denispop9 I'm assuming the content is fine but we don't need all the styling that you've included, please review the changes in the original pr that is linked in the trello card which will show how the other files were added.